assets/xml/objects/object_ani.xml

```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="object_ani" Segment="6">
+        <Skeleton Name="gObjAniSkel" Type="Flex" LimbType="Standard" Offset="0xF0"/>
+        <Animation Name="gObjAniAnim0" Offset="0x67B8"/>
+        <Animation Name="gObjAniAnim1" Offset="0x70F0"/>
+        <Animation Name="gObjAniAnim2" Offset="0x76EC"/>
```

Comment:
The type suffix should always be last (except for offsets)
```suggestion
        <Animation Name="gObjAni0Anim" Offset="0x67B8"/>
        <Animation Name="gObjAni1Anim" Offset="0x70F0"/>
        <Animation Name="gObjAni2Anim" Offset="0x76EC"/>
```

Also, you could take a look at those animations in Z64Utils' Skeleton Viewer to know what are they and be able to name them properly.